app/src/ui/diff/image-diffs/two-up.tsx

```diff
@@ -27,40 +29,69 @@ export class TwoUp extends React.Component<ITwoUpProps, {}> {
+      this.props.current.bytes
+    )
+    const diffBytes = this.props.current.bytes - this.props.previous.bytes
 
     return (
-      <div className="image-diff-two-up" ref={this.props.onContainerRef}>
-        <div className="image-diff-previous">
-          <div className="image-diff-header">Deleted</div>
-          <ImageContainer
-            image={this.props.previous}
-            onElementLoad={this.props.onPreviousImageLoad}
-            style={style}
-          />
+      <div className="image-diff-container" ref={this.props.onContainerRef}>
+        <div className="image-diff-two-up">
+          <div className="image-diff-previous">
+            <div className="image-diff-header">Deleted</div>
+            <ImageContainer
+              image={this.props.previous}
+              onElementLoad={this.props.onPreviousImageLoad}
+              style={style}
+            />
 
-          <div className="image-diff-footer">
-            <span className="strong">W:</span> {previousImageSize.width}px |{' '}
-            <span className="strong">H:</span> {previousImageSize.height}px
+            <div className="image-diff-footer">
+              <span className="strong">W:</span> {previousImageSize.width}px |{' '}
+              <span className="strong">H:</span> {previousImageSize.height}px |{' '}
+              <span className="strong">Size:</span>{' '}
+              {formatBytes(this.props.previous.bytes)}
+            </div>
           </div>
-        </div>
 
-        <div className="image-diff-current">
-          <div className="image-diff-header">Added</div>
-          <ImageContainer
-            image={this.props.current}
-            onElementLoad={this.props.onCurrentImageLoad}
-            style={style}
-          />
+          <div className="image-diff-current">
+            <div className="image-diff-header">Added</div>
+            <ImageContainer
+              image={this.props.current}
+              onElementLoad={this.props.onCurrentImageLoad}
+              style={style}
+            />
 
-          <div className="image-diff-footer">
-            <span className="strong">W:</span> {currentImageSize.width}px |{' '}
-            <span className="strong">H:</span> {currentImageSize.height}px
+            <div className="image-diff-footer">
+              <span className="strong">W:</span> {currentImageSize.width}px |{' '}
+              <span className="strong">H:</span> {currentImageSize.height}px |{' '}
+              <span className="strong">Size:</span>{' '}
+              {formatBytes(this.props.current.bytes)}
+            </div>
           </div>
         </div>
+        <div className="image-diff-summary">
+          Diff:{' '}
+          <span
+            className={classNames({
+              added: diffBytes > 0,
+              removed: diffBytes < 0,
+            })}
+          >
+            {diffBytes !== 0
+              ? `${formatBytes(diffBytes, Sign.Forced)} | ${diffPercent}`
```

Comment:
Suggestion:  replace `| ${diffPercent}` with `(${diffPercent})`
The pipe is used earlier in this page to seperate unrelated informational units (e.g. width and height), while  absolute bytes and percentage are different ways to describe the same information.
Also because the pipe is rendered in the same green/red as the numbers next to it, it can be read as a `1` at first glance

Reply:
@jessetan I think it's nice suggestion, thanks, updated.